app/models/host.rb

```diff
@@ -17,7 +34,18 @@ class Host < ApplicationRecord
   sortable_by :os_minor_version, OS_MINOR_VERSION
   sortable_by(
     :ssg_version,
-    Arel.sql("string_to_array(benchmarks.version, '.')::int[]"),
+    Arel::Nodes::NamedFunction.new(
+      'CAST',
+      [
+        Arel::Nodes::NamedFunction.new(
+          'string_to_array',
+          [
+            Xccdf::Benchmark.arel_table[:version],
+            Arel::Nodes.build_quoted('.')
```

Comment:
I see, that somewhere we're also using `Arel::Nodes::Quoted`, not sure what's the advantage in which one :thinking: